docs/source/notebooks/nansat-introduction.ipynb

```diff
@@ -129,7 +129,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "n.write_map('map.png', pltshow=True)"
+    "n.write_figure('map.png', pltshow=True)"
```

Comment:
Originally it was `write_map`, bu now we have moved it to `tools.write_domain_map`. And `Nansat.write_figure` doesn't have pltshow, option.

Reply:
what does this mean? Is the code correct as it is, or not?